product_multi_image/models/product_product.py

```diff
@@ -31,6 +31,12 @@ def _compute_image_ids(self):
             )
             product.image_ids = [(6, 0, images.ids)]
 
+            product.image_1920 = (
```

Comment:
IIUC inside a compute function of the field `image_ids` we are assigning a value to a field (`image_1920`) that is not the field being computed (`image_ids`)? I guess I don't understand the problem, but it looks a bit fishy. Could you please expand on why this is needed (I did not find any mention in PR description or commit message)? Also, the `image_1920` itself [seems to be a computed](https://github.com/odoo/odoo/blob/f964b761eae32016e36dd9d39cc2c35fc2270eba/addons/product/models/product.py#L141) field.

Reply:
Ok, `image_1920` is in 14.0 main image of the product. It is computed (and stored in db) in standard. It takes its value from setting manually or from `product.template`. Now in `product_multi_image`, it should be taken from first image from multi image tab. Original approach was to assign it on `_inverse_image_ids` method, however it does not work in 100%:
```
product.image_1920 = (
                False if len(product.image_ids) < 1 else product.image_ids[0].image_main
            )
```
So my reasoning was, that it should be computed whenever `image_ids` are computed, so in `_compute_image_ids` method.

It quite feels like a hack but whenever I tried to suit original approach, it wouldn't work in all cases.

Reply:
So the goal is to change it so that `image_1920` is taken from `image_ids`? Then perhaps we should override `image_1920` [compute function](https://github.com/odoo/odoo/blob/f964b761eae32016e36dd9d39cc2c35fc2270eba/addons/product/models/product.py#L153)?

Reply:
Thanks for the suggestion. I tried that with no success. I suspect, the issue is again connected to changing binary value to its size, however this time I couldn't solve it with just changing context.